orga/tests/unit/components/auth/login-form_test.js

```diff
@@ -30,5 +36,65 @@ module('Unit | Component | Routes | login-form', (hooks) => {
       // then
       assert.ok(_authenticateStub.calledWith(sinon.match.any, expectedEmail));
     });
+
+    module('When there is no invitation', () => {
+      test('should not accept organization invitation when parameters are invalid', function (assert) {
```

Comment:
**suggestion**
`when parameters are invalid` -> `when form is invalid`

Reply:
done